client/asset/ltc/regnet_test.go

```diff
@@ -180,42 +180,43 @@ func TestWallet(t *testing.T) {
 
 	// Check available amount.
 	for name, wallet := range rig.backends {
-		available, unconf, err := wallet.Balance(tBTC.FundConf)
-		tLogger.Debugf("%s %f available, %f unconfirmed", name, float64(available)/1e8, float64(unconf)/1e8)
+		available, err := wallet.Balance([]uint32{tLTC.FundConf})
+		a := available[0]
```

Comment:
If `err != nil`, I would not expect `len(available)>0`.

Reply:
also added a check that `len(available)>0` although may not be necessary.